src/main/java/edu/harvard/iq/dataverse/util/PersonOrOrgUtil.java

```diff
@@ -73,6 +61,7 @@ public static JsonObject getPersonOrOrganization(String name, boolean organizati
 
         boolean isOrganization = !isPerson && Organizations.getInstance().isOrganization(name);
         if (!isOrganization) {
+            String[] orgPhrases = JvmSettings.ORG_PHRASE_ARRAY.lookupOptional(String[].class).orElse(new String[]{});
```

Comment:
It appears that the shift from a static block to looking these up as needed is just to be able to use the @JvmSetting annotation in tests? Putting the code here means these lookups will be done ~1M times for a /reExportAll command for Harvard Dataverse. Could/should we just call updated versions of the static methods in the tests?

Reply:
I agree that we should avoid making these lookups in normal operations, if at all possible. Sorry I missed this.

Reply:
@vera - can you revert the change here that and go back to having a static initialization and setX methods for use in testing? I'll move this to In Progress until you're finished.

Reply:
@qqmyers done :)